web-app/js/ng-app/book/controllers/book.controller.js

```diff
@@ -1,15 +1,18 @@
 angular.module('ang').controller('BookController', [
-    '$route',
-    'bookService',
+    '$state',
+    'BookService',
     BookController
 ]);
 
-function BookController($route, bookService) {
+function BookController($state, BookService) {
     var vm = this;
+
     vm.reverse = false;
 
-    bookService.getAllBooks().then(function (response) {
+    BookService.getAllBooks().then(function (response) {
         vm.books = response.data
+    }, function (response) {
```

Comment:
Use .catch instead.

Avoid console.log.. User doesn't see errors in console. Show them in the view.